public/config/configuration.json

```diff
@@ -9,7 +9,6 @@
   "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
   "LEGAL_API_URL": "https://legal-api-dev.apps.silver.devops.gov.bc.ca",
-  "LEGAL_API_VERSION": "/api/v1",
   "LEGAL_API_VERSION_2": "/api/v2",
```

Comment:
This is the name of the key in 1Password (our key repository). We have both the original and V2 keys because some projects still use both so we can't just change the value of the original.